api/src/main/java/com/velocitypowered/api/event/proxy/ServerUnRegisteredEvent.java

```diff
@@ -0,0 +1,25 @@
+package com.velocitypowered.api.event.proxy;
+
+import com.google.common.base.Preconditions;
+import com.velocitypowered.api.proxy.server.ServerInfo;
+
+/**
+ * This event is fired by the proxy before a backend server is unregistered
+ */
+public class ServerUnRegisteredEvent {
```

Comment:
Not a fan of this name personally, specifically the "UnRegistered" part

Reply:
Do you have any suggestions for what the name should be?

Reply:
lowercase r